Documentation/ceph-cluster-crd.md

```diff
@@ -117,7 +117,7 @@ Below are the settings available, both at the cluster and individual node level,
   - `^s`: Selects all devices that start with `s`
   - `^[^r]`: Selects all devices that do *not* start with `r`
 - `devices`: A list of individual device names belonging to this node to include in the storage cluster.
-  - `name`: The name of the device (e.g., `sda`).
+  - `name`: The name of the device (e.g., `sda`), or full udev path (e.g. `/dev/disk/by-id/ata-ST4000DM004-XXXX` - this will not change after reboots).
```

Comment:
Should this doc mention the new `fullPath` property, which would override the `name`?